css/button.css

```diff
@@ -0,0 +1,41 @@
+.previousbttn{
```

Comment:
BEM concepts not used, at least separate class name previousbttn to previous-btn